spec/migrations/20171117201519_update_resource_group_type_for_azure_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_migration
+
+describe UpdateResourceGroupTypeForAzure do
+  let(:resource_group_stub) { migration_stub :ResourceGroup }
+  let(:expected_type) { 'ManageIQ::Providers::Azure::ResourceGroup' }
+
+  migration_context :up do
```

Comment:
Add the corresponding `migration_context :down` test, too.

Reply:
Done.